src/coreclr/jit/emitriscv64.cpp

```diff
@@ -2112,20 +2115,615 @@ void emitter::emitJumpDistBind()
  *  Emit a 32-bit RISCV64 instruction
  */
 
-unsigned emitter::emitOutput_Instr(BYTE* dst, code_t code)
+unsigned emitter::emitOutput_Instr(BYTE* dst, code_t code) const
 {
     assert(sizeof(code_t) == 4);
     memcpy(dst + writeableOffset, &code, sizeof(code_t));
     return sizeof(code_t);
 }
 
+static void assertCodeLength(unsigned code, uint8_t size)
+{
+#ifdef DEBUG
```

Comment:
nit: isn't assert defined in PAL only for DEBUG already?

Reply:
I couldn't find a similar statement, maybe I missed one. Since this is a fairly simple function I think that this isn't an issue

Reply:
Yes this is unnecessary.

https://github.com/dotnet/runtime/blob/1303c3ad3239188afa55391c47512473ffae214a/src/coreclr/jit/host.h#L51

Reply:
Oh, I misunderstood the anwser. Ok I will remove this ifdefs